cli/commands/hcl/validate/cli.go

```diff
@@ -0,0 +1,76 @@
+package validate
+
+import (
+	"github.com/gruntwork-io/terragrunt/cli/commands/common/graph"
+	"github.com/gruntwork-io/terragrunt/cli/commands/common/runall"
+	"github.com/gruntwork-io/terragrunt/cli/flags"
+	"github.com/gruntwork-io/terragrunt/internal/cli"
+	"github.com/gruntwork-io/terragrunt/options"
+)
+
+const (
+	CommandName = "validate"
+
+	StrictFlagName         = "strict"
+	InputFlagName          = "input"
+	ShowConfigPathFlagName = "show-config-path"
+	JSONFlagName           = "json"
+)
+
+func NewFlags(opts *options.TerragruntOptions, prefix flags.Prefix) cli.Flags {
+	tgPrefix := prefix.Prepend(flags.TgPrefix)
+	terragruntPrefix := flags.Prefix{flags.TerragruntPrefix}
+	terragruntPrefixControl := flags.StrictControlsByCommand(opts.StrictControls, CommandName)
+
+	flagSet := cli.Flags{
+		flags.NewFlag(&cli.BoolFlag{
+			Name:        StrictFlagName,
+			EnvVars:     tgPrefix.EnvVars(StrictFlagName),
+			Destination: &opts.HCLValidateStrict,
+			Usage:       "Enables strict mode. <explanation of what strict mode means>",
```

Comment:
Really? 😂 

Please update this with an actual explanation.

Reply:
Maybe a good explanation is:

```go
			Usage:       "Enables strict mode. When used in combination with the `--inputs` flag, any inputs defined in Terragrunt that are _not_ used in OpenTofu/Terraform will trigger an error.",
```

Reply:
Ah, I didn't pay attention when I clicked through GitHub ui "Commit changes". Thanks, updated.